dashboard/test/helpers/survey_results_helper_test.rb

```diff
@@ -0,0 +1,54 @@
+require 'test_helper'
+
+class SurveyResultsHelperTest < ActionView::TestCase
+  include SurveyResultsHelper
+
+  def sign_in(user)
+    # override the default sign_in helper because we don't actually have a request or anything here
+    stubs(:current_user).returns user
+  end
+
+  setup do
+    @teacher = create(:teacher, created_at: 3.weeks.ago)
+    sign_in @teacher
+  end
+
+  test 'teacher account existed 14 days' do
+    assert account_existed_14_days?
+  end
+
+  test 'teacher account newer than 14 days' do
+    @teacher.update(created_at: 1.week.ago)
+    refute account_existed_14_days?
+  end
+
+  test 'teacher account has students' do
+    create :follower, user: @teacher
+    assert has_any_students?
+  end
+
+  test 'teacher account no students' do
+    refute has_any_students?
+  end
+
+  test 'teacher account has no student under 13' do
+    create :follower, user: @teacher
```

Comment:
Please don't rely implicitly on `FactoryGirl` giving you a `follower` with a `young_student`. Prefer something like `create :follower, user: @teacher, student_user: (create :young_student)`.